src/pipeline/config.cpp

```diff
@@ -16,7 +16,7 @@ namespace librealsense
         {
             std::lock_guard<std::mutex> lock(_mtx);
             _resolved_profile.reset();
-            _stream_requests[{stream, index}] = { stream, index, width, height, format, fps };
+            _stream_requests[{stream, index}] = { stream, index, width, height, fps, format };
```

Comment:
Is it a bug fix ?

Reply:
I removed `struct request_type` and replaced it with `stream_profile`. the fps and format are in different locations.